channels/qa.m3u

```diff
@@ -58,3 +58,7 @@ https://qatartv.akamaized.net/hls/live/2026573/qtv1/master720p.m3u8
 https://qatartv.akamaized.net/hls/live/2026574/qtv2/master.m3u8
 #EXTINF:-1 tvg-id="QatarTV2.qa" tvg-name="Qatar TV 2" tvg-country="QA" tvg-language="" tvg-logo="" group-title="",Qatar TV 2 (720p)
 https://qtv-i.akamaihd.net/hls/live/1020532/QTV2/master.m3u8
+#EXTINF:-1 tvg-id="BanDaoDianShiTai.qa" tvg-name="半岛电视台" tvg-country="QA" tvg-language="" tvg-logo="" group-title="",半岛电视台
```

Comment:
That's Al Jazeera Balcans, a link that I have removed as a duplicate

Reply:
Removed.